semantic/src/definitions/modules/accordion.less

```diff
@@ -118,9 +118,9 @@
      Styled
 ---------------*/
 
-.ui.styled.accordion {
-  width: @styledWidth;
-}
+// .ui.styled.accordion {
+//   width: @styledWidth;
+// }

```

Comment:
I really didn't want to have to edit this file (we should be using just the "overrides" file). But I couldn't find a way to get rid of the "width" property (I tried `unset` and `revert` but neither worked properly). 

So we get rid of the fixed width for the accordions and rely on a max-width based on context.

Reply:
I also found I couldn't 'override' modules the other day...  I had a quick look at how to fix, but decided to move on.